### PR TITLE
fix(filter): edit target short code display names for filter

### PR DESCRIPTION
**Pull request by danvanhorn**
# Changes introduced
I've created a translation proccess by which we can change the displayed values for our filter. We will be matching the text of the filter in sample items website as much as possible and adhere to the Excel spreadsheet sent to us by smarter balanced.

## Contributor checklist

- [x] Wrote/updated tests for introduced changes
- [x] ~~Added new stories for created/modified components (if applicable)~~
- [x] Updated Postman library for created/modified routes (if applicable)
- [x] Verified that there are no issues in CircleCI
- [x] Assigned yourself to the PR
- [x] Removed `WIP:` from the PR title
- [x] Requested review from the SB Reviewers team

## Reviewer checklist

- [ ] Assigned yourself to the PR
- [ ] Read through the changes in the `Files changed` tab
- [ ] Verified that tests were written/modified
- [ ] ~~Verified that stories were written/modified (if applicable)~~
- [ ] Verified that Postman was updated (if applicable)
- [ ] Checked out the branch and tested changes locally

### CODE REVIEW

**Comment by danvanhorn** [comment]
We will be expecting undefined to be returned for the description for all targets under the ELA subject. This is because the claim numbers are not quite correct in the db

File: src/dal/helpers/index.ts [hunk 1]
```diff
@@ -46,7 +46,16 @@ export function buildSubjectsAndGrades(res: IGradeAndSubjectResult): IFilterOpti
   return subject.length !== 0 && grades.length !== 0 ? { subject, grades } : undefined;
 }
 
-export function buildClaimNumbers(dbResult: IClaimNumberResult[]): IFilterOptions | undefined {
+// function translateClaimNumber(subject: string, claimNumber: string): string {
+//   const claimNum: string = claimNumber[1];
+
+//   return subject === 'Math' ?  `${claimNum}: mathClaims[claimNumber]` : `${claimNum}: elaClaims[claimNumber]`;
+// }
```
Comment (puregarlic) [inline src/dal/helpers/index.ts:53]: Is there some reason to keep this comment around?

**Review by puregarlic — APPROVED** [review]
Awesome.

**Pull request merged**